src/terra-dev-site/app/demo/EventEmitter.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from 'react';
+import EventEmitter from 'terra-application/lib/utils/event-emitter';
+
+const EventEmitterExample = () => {
+  const [counter, setCounter] = useState(0);
+
+  useEffect(() => {
+    const listener = () => {
+      setCounter(prevCounter => prevCounter + 1);
+    };
+
+    EventEmitter.on('increment-counter', listener);
+
+    return () => {
+      EventEmitter.removeAllListeners();
```

Comment:
Can we change the example to turn off the specific event/handler? Anyone that copies this will remove all bound listeners (even the ones they didn't add) and be really confused.

```suggestion
      EventEmitter.off('increment-counter', listener);
```

Reply:
Fair enough: https://github.com/cerner/terra-application/pull/72/commits/7849a5af7ae295d8834691b474da133582ef304e